.github/workflows/deployment-script.yml

```diff
@@ -18,10 +18,10 @@ jobs:
             node-version: '12'
             check-latest: true
     - run: |
-        git clone -b pipeline-new https://github.com/virtual-labs/ph3-lab-mgmt
+        git clone --depth=1 https://github.com/virtual-labs/ph3-lab-mgmt -b dev
```

Comment:
Do we still need to clone the dev branch here? Please confirm with Pranav/Shlok that these changes are merged into the main branch.

Reply:
Pranav confirmed there is no need to clone the dev branch. Probably got pushed when Sravanthi ma'am was working on it. Updated now.